### DISCUSSION TITLE
What is the theme showed in the README.md video?

### DISCUSSION THREAD

**Discussion by shellwhale** [sayanarijit/xplr#722] [Q&A]
[](https://private-user-images.githubusercontent.com/11632726/166747867-8a4573f2-cb2f-43a6-a23d-c99fc30c6594.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE2NTEwMDAsIm5iZiI6MTcyMTY1MDcwMCwicGF0aCI6Ii8xMTYzMjcyNi8xNjY3NDc4NjctOGE0NTczZjItY2IyZi00M2E2LWEyM2QtYzk5ZmMzMGM2NTk0Lm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIyVDEyMTgyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3MjNjYjJkNDU3NWIwOTUzNWI4MTkwNWVlZTdmNmQ4NjZiOTg2MjczNDIyZDc5MmM2MmYwMGVlMTdhODE5NGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.msg4tB101GPQS9_gl8WuAkvPBR8IBqh_ugw29Rcv7DA)

**Comment by sayanarijit** [comment]
I think it’s the default theme. Only icons plugin was used.

**Reply by shellwhale** [reply]
Oh okay, I think that my question was more which icon plugin is used

**Reply by sayanarijit** [reply]
I guess https://github.com/prncss-xyz/icons.xplr and https://github.com/dtomvan/extra-icons.xplr

Or just https://gitlab.com/hartan/web-devicons.xplr
Forgot actually...
